### PR TITLE
Add time requirement for Technical Assistant

### PR DESCRIPTION
**Pull request by leonardo-dabepis**
## About the PR

Added a time requirement for the Technical Assistant role.

## Why / Balance

There has been a common issue lately where griefers will take the Technical Assistant role so that they can have easy access to tools and easily sabotage power or welderbomb. Adding a short 4 hour requirement will deter griefers from doing that.

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Technical Assistant now has a 4 hour playtime requirement.

### CODE REVIEW

**Comment by Colin-Tel** [comment]
Why civilian hours? Why not just general playtime?

Not that I'm against this.

**Comment by leonardo-dabepis** [comment]
> Why civilian hours? Why not just general playtime?
> 
> Not that I'm against this.

I'll be honest I made this very quickly, but also they should suffer as a civilian before they get the right to use insuls

**Comment by Colin-Tel** [comment]
> I'll be honest I made this very quickly, but also they should suffer as a civilian before they get the right to use insuls

While I do think that is a fun idea, I think a guy playing as a medical intern should also be able to have the hours he put in our server itself count towards the novice role for another department.

**Comment by leonardo-dabepis** [comment]
> > I'll be honest I made this very quickly, but also they should suffer as a civilian before they get the right to use insuls
> 
> While I do think that is a fun idea, I think a guy playing as a medical intern should also be able to have the hours he put in our server itself count towards the novice role for another department.

oh, I didn't realize doctors aren't civilians :) 
I'll try to figure out how to change it to general playtime then, since that makes more sense.

**Comment by leonardo-dabepis** [comment]
> > I'll be honest I made this very quickly, but also they should suffer as a civilian before they get the right to use insuls
> 
> While I do think that is a fun idea, I think a guy playing as a medical intern should also be able to have the hours he put in our server itself count towards the novice role for another department.

I have now changed it to an overall playtime requirement of 4 hours.

File: Resources/Prototypes/Roles/Jobs/Engineering/technical_assistant.yml [hunk 1]
```diff
@@ -1,10 +1,12 @@
-﻿- type: job
+- type: job
```
Comment (Colin-Tel) [inline Resources/Prototypes/Roles/Jobs/Engineering/technical_assistant.yml:1]: I'm just trying to see how this line was even touched...

Comment (Colin-Tel) [reply Resources/Prototypes/Roles/Jobs/Engineering/technical_assistant.yml:1]:
```suggestion
- type: job 
```
oh there's supposed to be a space after it

Comment (Colin-Tel) [reply Resources/Prototypes/Roles/Jobs/Engineering/technical_assistant.yml:1]: wait shit no there isn't what the hell man

Comment (Colin-Tel) [reply Resources/Prototypes/Roles/Jobs/Engineering/technical_assistant.yml:1]: Fuck it we ball

**Review by Colin-Tel — APPROVED** [review]

**Pull request merged**